code/__DEFINES/mobs.dm

```diff
@@ -202,4 +202,4 @@
 
 // AI Toggles
 #define AI_CAMERA_LUMINOSITY	5
-#define AI_VOX // Comment out if you don't want VOX to be enabled and have players download the voice sounds.
+#define AI_VOX // Comment out if you don't want VOX to be enabled and have players download the voice sounds.
```

Comment:
??

There is nothing in this file in this commit besides removing the line ending.

Reply:
there WAS something in this file, i removed it, now git is being a shit